.travis.yml

```diff
@@ -8,33 +8,28 @@ addons:
   apt:
     packages:
     - s3cmd
-    - ruby
-    
+
 notifications:
   email: false
   slack:
     rooms:
       - gapminder:siB4Z9ymsYR6qHnRPpgUoB2Q#vizabi-spam
     on_success: change
     on_failure: always
-  
+
 cache:
   directories:
     - node_modules
 
 before_install:
   - npm install -g npm@latest
-  - gem install sass
-  - gem install scss_lint
-  - npm install -g gulp
 
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-  - gulp &
+  - npm run build:prod &
```

Comment:
- npm run build:prod
it should be without `&`, you need to actually wait until build will be finished